docs/DeveloperGuide.md

```diff
@@ -309,7 +349,7 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
+* has a need to manage a significant number of entries
```

Comment:
in this case should we say expenditure and income instead?

Reply:
Resolved